new/LIT7097SanaSom.xml

```diff
@@ -0,0 +1,87 @@
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <listWit>
+                    <witness corresp="UppQ1524o"/>
+                </listWit>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                    <p>Poetic composition, defined in the manuscript as a <foreign xml:lang="gez">Gubāʾa Qānā</foreign>.
+                        However, no rhymes can be discerned, as one would expect in a <foreign xml:lang="gez">Gubāʾa Qānā</foreign>-type
+                        <foreign xml:lang="gez">qəne</foreign>. 
+                        The composition has no author attribution in the manuscript.</p>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="ChristianLiterature"/>
+                    <term key="Poetry"/>
+                </keywords>
+            </textClass>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="JK" when="2024-10-11">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="bibliography">
+                <listRelation>
+                    <relation name="lawd:hasAttestation" active="LIT7097SanaSom" passive="UppQ1524o"/>
+                </listRelation>
+            </div>
+            <div type="edition" xml:lang="gez">
+                <note>This edition is based on <ref type="mss" corresp="UppQ1524o"/> on fols 4r, 5r.
+                    Division into verses is indicated with ፤ .
+                    The end of the poem, as well as some poem-internal verse divisions, are indicated with ። .
+                    Line 5 is indented.
+                </note>
+                <ab>
+                    <l n="1">ጸንአ፡ ጾም፡ <sic resp="JK">ነግረ፡</sic> ኢሳይያ፡ አብ፤</l>
+                    <l n="2">አምጣነ፡ ባቲ፡ ወአሉ፤</l>
+                    <l n="3">እስከ፡ ፍና፤</l>
+                    <l n="4">ሰርክ፡ ሕዝብ፤</l>
+                    <l n="5">መፍትው።</l>
+                    <l n="6">ይትፌሳሕ፡ ቀኖና፡ ሥጋ፤</l>
+                    <l n="7">በጽርሁ፡ ቀርነ፡ እንስሳ፥ እልው።</l><pb n="5r"/>
+                    <l n="8">እስመ፡ ተኃንቀ፡ ጉርኤሁ።</l>
+                    <l n="9">ዮም፡ ዕለት፡ ብዙሕ፡ ትፍስሕት፤</l>
+                    <l n="10">እስመ፡ በደብረ፡ ታቦር፤</l>
+                    <l n="11">ቆመ፡ ፈጣሪ፤</l>
+                    <l n="12">ወበደብረ፡ ብርሃን፡ ዳዊት።</l>
```

Comment:
Because it is Guba'e Qana, it should be given in two verses (`<lb>`) ending in the rhyme አሉ/እልው.

Reply:
Very interesting! Ok, then I divide it up into two, ok?

I can't find a LIT ID for the qene you mention, Denis, so I will go ahead and create one for it (with the Uppsala manuscript as a manuscript witness and the printed book as an edition)

Reply:
It is not in Bm yet, here is a portion of the book, see no. 246 (the excerpt has no title, I have the book at home and will check). 
[library63a8.pdf](https://github.com/user-attachments/files/17362678/library63a8.pdf)

Reply:
Thank you!

Reply:
And `poem3 `(`corresp `is correct, thank you) contains I guess
line 4 (formerly 9)
line 5 (formerly 10-12)

Reply:
Still I would divide the poems, because in work records we treat the works irrespective of the way they are shaped in the manuscripts. We assume that both poems were created as separate compositions and only by chance were copied together, with an obscure line in between (unless it is an incredible case of "double qene"). If the structure of the written lines do not fit the structure prescribed for the qene type, this is the phenomenon of the text arrangement in the manuscript. In other words, the qene ጸንአ፡ ጾም፡ ... fits guba'e qana type, but it's two long verses are simply not copied in two lines, by whatever reasons. But for us it is absolutely justified to present the poem in its work ID as composed of two lines (መፍትው - እልው), and (provisionally) assigned a type to it. If you want to demonstrate its shape in the manuscript, you transcribe the poem in the same structure in the record for manuscript (then, division in verses/lines and written lines will be as you indicated here). In a similar way, we do not reproduce rubrications in work ID - all this pertains to shaping the text in concrete witnesses and should be treated in manuscript records. Also, the fact that one of the qenes is recorded in a publication is another argument for separating them.

Reply:
As I understand work records, we want to record works that are circulating in the Ethiopian manuscript culture. I agree with your analysis that what we find in the Uppsala manuscript must originally have been two guba'e qana qenes plus one spurious line – but the manuscript has what it has, which is clearly something analysed (probably wrongly! :) ) as one work.

I have a suggestion: would it be OK if I create a separate ID for the first, hypothetical qene (the two lines መፍትው - እልው), parallel to the one I created for Yom 3elätät. And then I still keep one "general record" for the entire text as it is attested in the Uppsala manuscript, i.e. the two qenes plus the spurious line. Otherwise, I would need to create a separate LIT ID for the spurious line as well, and it feels like that is taking it too far (and it also departs farther and farther away from how we define a work, doesn't it?)

Reply:
Creating ID for the first qene and a separate ID for the entire block is ok, but you are not obliged to create an ID for one line, because it is not a work. We do not know exactly how the passage was understood by the writer. Even though, the way it is copied is the phenomenon of the transmission in concrete witnesses.  If you work in manuscripts, you will find a lot of "noise" - separate lines and small text bits, which are not works (they may be parts of some works, but to trace back everything is not realistic).